tests/s2_inference/test_encoding.py

```diff
@@ -149,3 +149,26 @@ def test_model_un_normalization(self):
 
                 assert abs(max_output_norm - 1) > eps, f"{name}, {sentence}"
                 assert abs(min_output_norm - 1) > eps, f"{name}, {sentence}"
+
+    def test_open_clip_vectorize(self):
```

Comment:
can we also add a test that checks the output embedding dimension against the one in the registry?